src/wasm-binary.h

```diff
@@ -1255,7 +1261,7 @@ class WasmBinaryWriter {
   void writeFunctions();
   void writeGlobals();
   void writeExports();
-  void writeDataCount();
+  void writeDataSegmentCount();
```

Comment:
The name of the section this writes is the "data count section," so I don't think the name needs to change.